homework/schedule/CMakeLists.txt

```diff
@@ -8,8 +8,8 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 include_directories(.)
```

Comment:
We should also change paths to `#include "catch/catch_amalgamated.hpp"` in `tests.cpp` and `tests-bonus.cpp` or add `catch` to `include_directories`.

I prefer the first option.